vagrant/provisioning/roles/arkcase-file-utils/tasks/main.yml

```diff
@@ -75,24 +75,24 @@
       dest: "/tmp/LibreOffice_{{ libreoffice_version | default('7.3.4') }}_Linux_x86-64_install-rpm_en-US.tar.gz"
       owner: root
       url: "https://ftp.osuosl.org/pub/tdf/libreoffice/stable/{{ libreoffice_version | default('7.3.4') }}/rpm/x86_64/LibreOffice_{{ libreoffice_version | default('7.3.4') }}_Linux_x86-64_rpm.tar.gz --no-check-certificate"
-  when: not libreoffice_bin_exists
+  when: libreoffice_bin_exists
```

Comment:
What is your purpose with this change?  I'm not sure how the code worked, it seems the check should be more like: `when: not libreoffice_bin_exists.stat.exists`, see https://docs.ansible.com/ansible/latest/collections/ansible/builtin/stat_module.html#return-values

Reply:
and same comment for all the other changes.

Reply:
I removed the when because I found out Ansible automatically skips the tasks when Libreoffice is already installed.